src/compiler.ts

```diff
@@ -7989,9 +7989,17 @@ export class Compiler extends DiagnosticEmitter {
         );
       }
       case ElementKind.FUNCTION_PROTOTYPE: {
+        let prototypeInstance = <FunctionPrototype>target;
```

Comment:
Typically named just "prototype" in other parts of the compiler.

Reply:
All set!